Port more cloud docs (#2684)
Qiskit/documentation: docs/guides/manage-cost.mdx

---
title: Manage cost on the Standard Plan

> description: How to manage costs of running jobs on QPUs when using the Qiskit Runtime Standard plan.

Reviewer: ```suggestion
description: How to manage costs of running jobs on QPUs when using the Standard Plan on IBM Quantum Platform.
```

---

# Manage cost on the Standard Plan

<LegacyContent>
<Admonition type="note">
This documentation is relevant to IBM Quantum&reg; Platform Classic. If you need the newer version, go to the new [IBM Quantum Platform documentation.]( https://quantum.cloud.ibm.com/docs/guides/manage-cost)
</Admonition>
</LegacyContent>

<CloudContent>
<Admonition type="note">
This documentation is relevant to the new IBM Quantum&reg; Platform. If you need the previous version, return to the [IBM Quantum Platform Classic documentation.](https://docs.quantum.ibm.com/guides/manage-cost)
</Admonition>
</CloudContent>

<LegacyContent>
The IBM Cloud&reg; Quantum Standard plan is not free, except when running jobs on simulators. Use the information in this topic to help you understand how much you’re paying and how to limit your costs.

<Admonition type="note">

>   The information in this topic only applies to those who are using the Standard plan for Qiskit Runtime on IBM Cloud.  There are no costs associated with the IBM Quantum Platform Open Plan.

Reviewer: ```suggestion
  The information in this topic only applies to those who are using the Standard Plan for Qiskit Runtime on IBM Cloud.  There are no costs associated with the IBM Quantum Platform Open Plan.
```

</Admonition>

For instructions, see the [Manage costs](https://cloud.ibm.com/docs/quantum-computing?topic=quantum-computing-cost) topic in the IBM Cloud documentation.

Learn how to limit your costs when using the Standard Plan on IBM Quantum&reg; Platform.

<Admonition type="note">
  The information in this topic only applies to those who are using the Standard Plan for Qiskit Runtime.  There are no costs associated with the IBM Quantum Platform Open Plan.
</Admonition>
</LegacyContent>
<CloudContent>
Learn how to limit your costs when using the Standard Plan on IBM Quantum&reg; Platform.

<Admonition type="note">
  The information in this topic only applies to those who are using the Standard Plan for Qiskit Runtime.  There are no costs associated with the IBM Quantum Platform Open Plan.
</Admonition>
</CloudContent>
## Set a cost limit

An instance administrator can limit how much is spent.

> To set cost limits on a Standard Plan instance, navigate to the [IBM Cloud Instances page](https://cloud.ibm.com/quantum/instances), then click the instance and set the **Cost limit**. The cost limit refers to the total cost of all jobs run with this instance since it was created, and it will always be greater than or equal to the Total cost. After the instance reaches the specified number of total seconds, no further jobs can be run and no more cost is incurred.

Reviewer: ```suggestion
To set cost limits on a Standard Plan instance, navigate to the [IBM Cloud Instances page](https://cloud.ibm.com/quantum/instances), then click the instance and set the **Cost limit**. The cost limit refers to the total cost of all jobs run with this instance since it was created, and it will always be greater than or equal to the total cost. After the instance reaches the specified number of total seconds, no further jobs can be run and no more cost is incurred.
```
